src/discord/commands/setup/streams/disable.ts

```diff
@@ -0,0 +1,30 @@
+import { Declare, Group, SubCommand, type CommandContext } from "seyfert";
+import { Guilds } from "../../../../models/Guilds";
+
+@Declare({
+    name: "disable",
+    description: "Disable Twitch notifications.",
+})
+@Group("streams")
+export default class ChannelCommand extends SubCommand {
+    async run(context: CommandContext) {
+        const data = await Guilds.findOneAndUpdate(
+            { guildId: context.guildId },
+            { $set: { "streams.notify": false } },
+            { new: true }
+        );
+
+        const profile = context.chatbot.profiles.get(data?.twitch.userId!);
+
+        if (profile) {
+            await profile.removeEvent("StreamOffline");
+            await profile.removeEvent("StreamOnline");
+        }
+
+        await context.editOrReply({

```

Comment:
Add ephemeral flag here + change response to `Notifications were disabled for live streams`